core/templates/core/clients.html

```diff
@@ -18,6 +18,7 @@
 {% block extra_js %}
     <script type="riot/tag" src="{% static 'core/tags/clients.tag' %}"></script>
     <script type="riot/tag" src="{% static 'core/tags/client.tag' %}"></script>
+    <script type="riot/tag" src="{% static 'core/tags/pager.tag' %}"></script>
```

Comment:
Having to do this in each template seems less than ideal, but I figured we don't want to *always* include the pager tag. Other ways to approach?